### PR TITLE
Deploy to S3

### PR DESCRIPTION
**Pull request by bjweaver**
Deploy to S3

### CODE REVIEW

File: .github/workflows/prod.yml [hunk 1]
```diff
@@ -1,69 +1,114 @@
...
 on:
-  push:
-    tags:
-      - prod
-
+  workflow_dispatch:
 jobs:
-  main:
+  build_eth:
     runs-on: ubuntu-latest
```
Comment (rdbarrios90) [inline .github/workflows/prod.yml:6]: I will add  "if: github.ref == 'refs/heads/main'" to ensure job executed only on mian branch

File: .github/workflows/prod.yml [hunk 2]
```diff
@@ -1,69 +1,114 @@
...
+
+      - name: Upload environment to be used by the next job
+        uses: actions/upload-artifact@v2
+        with:
+          name: build_eth
+          path: build
+          retention-days: 1
+  
+  build_bsc:
+    runs-on: ubuntu-latest
```
Comment (rdbarrios90) [inline .github/workflows/prod.yml:37]: I will add  "if: github.ref == 'refs/heads/main'" to ensure job executed only on mian branch

File: .github/workflows/prod.yml [hunk 3]
```diff
@@ -1,69 +1,114 @@
...
-          tag-custom: ${{ steps.slug.outputs.sha8 }}
-          tag-custom-only: true
+          name: build_bsc
+          path: build
+          retention-days: 1
 
-      - name: Configure AWS credentials
+  
+  push_eth:
+    needs: build_eth
```
Comment (rdbarrios90) [inline .github/workflows/prod.yml:69]: I will add  "if: github.ref == 'refs/heads/main'" to ensure job executed only on mian branch

File: .github/workflows/prod.yml [hunk 4]
```diff
@@ -1,69 +1,114 @@
...
-            REACT_APP_TOKEN_AUTH=${{ secrets.REACT_APP_TOKEN_AUTH_PROD}}
-            REACT_APP_SHOULD_FALLBACK=false
+      - name: Invalidate Cache
+        run: |
+          aws cloudfront create-invalidation --paths "/*" --distribution-id ${{ secrets.AWS_CLOUDFRONT_ID_ETH_PROD }}
 
-      - name: Push api
-        uses: ianbelcher/eks-kubectl-action@master
+  push_bsc:
+    needs: build_bsc
```
Comment (rdbarrios90) [inline .github/workflows/prod.yml:94]: I will add  "if: github.ref == 'refs/heads/main'" to ensure job executed only on mian branch

**Review by rdbarrios90 — COMMENTED** [review]
Add a check to ensure the workflow_dispatch in prod only is executed using main branch

**Review by leossantiago — APPROVED** [review]

**Review by rdbarrios90 — APPROVED** [review]

**Pull request merged**